test/index.js

```diff
@@ -1,257 +1,415 @@
 const sinon = require('sinon');
 const proxyquire = require('proxyquire').noCallThru();
-const nock = require('nock')
-const expect = require('chai').expect;
-const chromdriver = require('chromedriver');
-
-const originalProcessStdout = process.stdout;
-const originalProcessStderr = process.stderr;
-const originalConsoleLog = console.log;
-const originalConsoleError = console.error;
-
-let mockStdoutChunk;
-
-const mockSpawn = {
-  stdout: {
-    on(evt, callback) {
-      if (mockStdoutChunk) {
-        callback('Google Chrome 97.0.4692.71');
-        mockStdoutChunk = null;
-      }
-    }
-  },
-  stderr: {
-    on: sinon.stub()
-  },
-  on(event, callback) {
-    callback();
-  }
+const path = require('path');
+const fsPromises = require('fs').promises;
+const os = require('os');
+const chai = require('chai');
+const { capitalize } = require('../src/utils');
+const chaiAsPromised = require('chai-as-promised');
+const expect = chai.expect;
+chai.use(chaiAsPromised);
+
+const mockConsoleLog = sinon.stub(console, 'log');
```

Comment:
I suggest not mocking the console log like this as it makes debugging tests impossible. You'll forgot you did this then try to add console logs to the tests to see what's going on and nothing will output.

Instead I suggest [mocking the console in each test](https://github.com/straker/browser-driver-manager/blob/main/test/index.js#L217) where it's needed and only at the time it's needed. That way it's very obvious when we're mocking and we'll see that in the test and know how to work around it.